driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/gridfs/GridFSPublisherSpecification.groovy

```diff
@@ -167,6 +167,32 @@ class GridFSPublisherSpecification extends FunctionalSpecification {
                                            ByteBuffer.wrap(contentBytes)])
     }
 
+    def 'should upload multiple chunks when smaller than chunksize'() {
```

Comment:
Can you explain the intent of this test, as I'm not clear where multiple chunks come in to play.  On line 184 the test asserts that the chunks collection has only one document, and `contentBytes` is just six characters.

Reply:
Renamed the test to be more accurate as to the intent: 

"should upload multiple parts from the source publisher when total size is smaller than chunksize"